src/Bundle/Maker/Factory/AbstractDoctrineDefaultPropertiesGuesser.php

```diff
@@ -11,24 +9,23 @@
 /** @internal  */
 abstract class AbstractDoctrineDefaultPropertiesGuesser implements DefaultPropertiesGuesser
 {
-    public function __construct(protected ManagerRegistry $managerRegistry, private FactoryFinder $factoryFinder)
+    public function __construct(protected ManagerRegistry $managerRegistry, private FactoryFinder $factoryFinder, private FactoryGenerator $factoryGenerator)
     {
     }
 
     /** @param class-string $fieldClass */
-    protected function addDefaultValueUsingFactory(MakeFactoryData $makeFactoryData, string $fieldName, string $fieldClass, bool $isMultiple = false): void
+    protected function addDefaultValueUsingFactory(MakeFactoryData $makeFactoryData, MakeFactoryQuery $makeFactoryQuery, string $fieldName, string $fieldClass, bool $isMultiple = false): void
     {
         if (!$factoryClass = $this->factoryFinder->getFactoryForClass($fieldClass)) {
-            $makeFactoryData->addDefaultProperty(\lcfirst($fieldName), "null, // TODO add {$fieldClass} type manually");
-
-            return;
+            $factoryClass = $this->factoryGenerator->generateFactory($fieldClass, $makeFactoryQuery);
```

Comment:
so, here, when we don't know about a factory, we create it.

I'm wondering if we shouldn't ask the use if he/she really wants to create it?
This would mean we should bring the `$io` till here

Reply:
Yeah, we should probably confirm (unless they passed `--all`)

Reply:
done :)